homeassistant/components/saunum/config_flow.py

```diff
@@ -0,0 +1,79 @@
+"""Config flow for Saunum Leil Sauna Control Unit integration."""
+
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from pysaunum import SaunumClient, SaunumException
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+from homeassistant.const import CONF_HOST
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import config_validation as cv
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_HOST): cv.string,
+    }
+)
+
+
+async def validate_input(
+    hass: HomeAssistant, data: dict[str, Any], flow: ConfigFlow
```

Comment:
hass is unused btw, just like flow. I think you could even inline it at this point